### PR TITLE
chore(models): move from django-json-field to jsonfield for Django 1.9 compatibility

### PR DESCRIPTION
**Pull request by helgi**
Does not change any functionality and helps with other PRs

### CODE REVIEW

**Comment by slack** [comment]
Wherein the pythongs ( :snake: ) create unreadable migrations.

**Comment by helgi** [comment]
whaaaaaaaaa! it's just an up and a down and the whole schema defintion all in one ... oh yeah, unreadable :cry:

**Comment by bacongobbler** [comment]
what's the reason for this migration? Is Django JSON Field incompatible with Django v1.9 or something? Shouldn't we just contribute fixes upstream for 1.9 compatibility instead?

**Comment by helgi** [comment]
Yeah, it's incompatible - This seemed quicker as it provides the same functionality. The migrations are just so Django knows how to reference class it's hooked up to

https://www.djangopackages.com/grids/g/json-fields/ going by this as well the one we currently are using hasn't been touched since 2014 and is not Python 3 compatible.

https://github.com/derek-schaefer/django-json-field/issues/42 is the reason for the 1.9 compatibility problem. There is a PR, no dice tho.

**Comment by technosophos** [comment]
Do we want this in alpha1?

**Comment by helgi** [comment]
I feel it is as much of a risk as any other PR around. It can wait until Beta if people want to but I find it to be low risk unless we don't trust our tests (which have not always been able to catch pretty basic things, mind you)

**Comment by technosophos** [comment]
I'll leave it to @helgi @bacongobbler and @slack to decide whether we want to merge this today.

**Comment by bacongobbler** [comment]
I'd rather leave this out since it's not a showstopper for alpha, but I would like to get this one in as soon as we're done with the release.

**Comment by slack** [comment]
@helgi this ready to rumble?

**Comment by helgi** [comment]
This one is ready, just needs another LGTM

File: rootfs/api/south_migrations/0026_auto__chg_field_config_tags__chg_field_config_values__chg_field_config.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        u'contenttypes.contenttype': {
+            'Meta': {'ordering': "('name',)", 'unique_together': "(('app_label', 'model'),)", 'object_name': 'ContentType', 'db_table': "'django_content_type'"},
+            'app_label': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'model': ('django.db.models.fields.CharField', [], {'max_length': '100'}),
+            'name': ('django.db.models.fields.CharField', [], {'max_length': '100'})
+        }
+    }
+
+    complete_apps = ['api']
```
Comment (arschles) [inline rootfs/api/south_migrations/0026_auto__chg_field_config_tags__chg_field_config_values__chg_field_config.py]: for my knowledge, is this auto generated, and if so, by what?

Comment (helgi) [inline rootfs/api/south_migrations/0026_auto__chg_field_config_tags__chg_field_config_values__chg_field_config.py]: `cd rootfs && ./manage.py makemigrations api --auto` - this one is `South` that handles migrations

Comment (arschles) [inline rootfs/api/south_migrations/0026_auto__chg_field_config_tags__chg_field_config_values__chg_field_config.py]: got it, thanks

**Pull request merged**